deeds-dapp-webapp/src/main/webapp/vue-app/layout/components/Topbar.vue

```diff
@@ -60,6 +65,9 @@ export default {
     isTestNetwork() {
       return this.networkId === 4;
     },
+    isMobile() {
+      return this.$vuetify.breakpoint.smAndDown;
+    },
```

Comment:
You are using `VueX`, thus you can factorize this variable by adding it inside a shared variable in `main.js` instead of redefining it in each Vue component

Reply:
fixed